### ISSUE TITLE
db.get fails if zero is the value

### ISSUE THREAD

**Issue by luca-aurelia** [juliangruber/json-store#1]
To reproduce:

```
const JSONStore = require('json-store')
const db = JSONStore('./index.json')

db.set('zero', 0)
db.get('zero') // => returns undefined, but should return 0
```

The problem is [this line](https://github.com/juliangruber/json-store/blob/6e53a83af1e61fec9fa5dfcf19f161df1cf0d61b/index.js#L11). `!this.Store['zero']` evaluates to `!0`, which evaluates to **`true`**. As a result, the method returns without a return value.

**Comment by juliangruber** [comment]
happy to merge a patch :)

**Comment by luca-aurelia** [comment]
Happy to provide a patch :) This was fixed by merging pull request #4.
